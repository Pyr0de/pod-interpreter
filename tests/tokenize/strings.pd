"simple"
"space1 space2 space3"
"quotes '' ' "
'single quotes'
"fails
'fails again
