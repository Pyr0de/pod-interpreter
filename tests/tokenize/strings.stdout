ExitCode: 65
STRING "simple" simple 1
STRING "space1 space2 space3" space1 space2 space3 2
STRING "quotes '' ' " quotes '' '  3
STRING 'single quotes' single quotes 4
STRING "fails fail 5
STRING 'fails again fails agai 6
EOF  null
